api/v1/helpers/verbs/utils.js

```diff
@@ -680,7 +681,7 @@ function forbidden(next, modelName) {
 // ----------------------------------------------------------------------------
 
 module.exports = {
-
+  logAPI,
```

Comment:
only a few files would be free from logAPI: most files in api/v1/helpers/verbs with the name do*.js would include logAPI.